corehq/apps/repeaters/models.py

```diff
@@ -535,13 +536,19 @@ def fire(self, force_send=False):
     def post(self, post_info, tries=0):
```

Comment:
I'd added a few comments in my original commit that might be useful here. Also, it looks like you are only adding the `operation` property to the NikshayRepeater rather than the parent class. What's your rationale behind that? I think we might want to be able to use that later for other repeaters as it is generic behaviour.